### PR TITLE
init logging

### PR DESCRIPTION
**Pull request by gentlerainsky**
Try to implement #331
- log matplotlib backend

**PR In progress - need some idea on this**
e.g.
- What would you like to see in log?

### CODE REVIEW

File: arviz/__init__.py [hunk 1]
```diff
@@ -1,8 +1,20 @@
 # pylint: disable=wildcard-import,invalid-name,wrong-import-position
 """ArviZ is a library for exploratory analysis of Bayesian models."""
 __version__ = "0.2.1"
+from matplotlib import get_backend as mpl_get_backend
```
Comment (canyon289) [inline arviz/__init__.py]: I dont think we should include this import for two reasons. One is that matplotlib should handle its own logging, the other is that this adds another object to the namespace

**Comment by canyon289** [comment]
@gentlerainsky Thanks for submitting this. I agree with @ahartikainen's comments and left one inline.

**Comment by canyon289** [comment]
@gentlerainsky are we holding you up on this pull request?

If you're looking for some ideas there's print statements in `io_cmdstan.py` and `conftest.py` that would be well suited to convert to logging

**Comment by gentlerainsky** [comment]
@canyon289 @ahartikainen  Sorry that I'm quite for sometime. I've update the logging as you've suggested and I took the style from https://github.com/requests/requests/blob/master/requests/__init__.py#L128

For prints statement, I found only a couple of 'print' in the repo. so I'll try those first.

**Comment by canyon289** [comment]
There's no rush, just wanted to be sure we weren't blocking you

**Comment by canyon289** [comment]
@gentlerainsky Would you be ok if we merged this? Trying to unblock an issue that would use this logging code!

https://github.com/arviz-devs/arviz/issues/389

**Comment by gentlerainsky** [comment]
@canyon289 Sorry. I've been occupied for the past few weeks. Yes, that would be better for now. Sorry again that I can't complete this.

**Comment by canyon289** [comment]
@gentlerainsky I think you did a fine job! Thank you so much for your contribution. Just asking to be sure. 

Will double check and likely merge tomorrow

Thank you again

**Comment by canyon289** [comment]
Merging, Thanks @gentlerainsky

**Pull request merged**